docs/terraform-provider-build.md

```diff
@@ -19,13 +19,47 @@ Download Terraform for IBM Power Systems from https://www.power-devops.com/terra
 Install latest go from https://golang.org/dl/ and set your GOPATH
 
 ```
-go get -u github.com/dmacvicar/terraform-provider-libvirt
-go get -u github.com/terraform-providers/terraform-provider-random
-go get -u github.com/terraform-providers/terraform-provider-ignition
-go get -u github.com/terraform-providers/terraform-provider-null
-cd $GOPATH/bin/ ; mkdir -p ~/.terraform.d/plugins/; 
-cp * ~/.terraform.d/plugins/
+mkdir -p ~/.terraform.d/plugins/
+```
+
+**terraform-provider-libvirt**: Please refer to the section below for instructions on installing the libvirt provider plugin. For [more information](https://github.com/dmacvicar/terraform-provider-libvirt#building-from-source).
+```
+git clone https://github.com/dmacvicar/terraform-provider-libvirt $GOPATH/src/github.com/dmacvicar/terraform-provider-libvirt
+cd $GOPATH/src/github.com/dmacvicar/terraform-provider-libvirt
+make install
+cp $GOPATH/bin/terraform-provider-libvirt ~/.terraform.d/plugins/terraform-provider-libvirt
```

Comment:
Did you check with latest libvirt tag? If latest tag is working then we should stick to it instead of master. What you think?
We could also restrict the version in code.

Reply:
Also every where you are using `make build/install` but no git checkout? It seems we are asking to build the default branch and just name it as version to avoid the version check. Defeats the whole purpose IMO.